install/updateVersion.sh

```diff
@@ -103,24 +109,35 @@ function check_git_status() {
 # Generated merge yaml files for easy installation
 function merge_files() {
   SRC=$TEMP_DIR/templates
-  AUTH_SRC=$SRC/istio-auth
-  ISTIO=$ROOT/install/kubernetes/istio.yaml
-  ISTIO_AUTH=$ROOT/install/kubernetes/istio-auth.yaml
-  ISTIO_AUTH_WITH_CLUSTER_CA=$ROOT/install/kubernetes/istio-auth-with-cluster-ca.yaml
+  DEST=$ROOT/install/kubernetes
+  ISTIO=$DEST/istio.yaml
+  ISTIO_CLUSTER_WIDE=$DEST/istio-cluster-wide.yaml
+  ISTIO_AUTH=$DEST/istio-auth.yaml
+
+#TODO remove 2 lines below once the e2e tests no longer look for this file
+  cp $SRC/istio-rbac-beta.yaml.tmpl $DEST/istio-rbac-beta.yaml
```

Comment:
this should not be cp, should "substitute $SRC/istio-rbac-beta.yaml.tmpl $DEST/istio-rbac-beta.yaml"
and ditto for all the other ones